examples/iio-monitor.c

```diff
@@ -122,11 +123,13 @@ static struct {
 	const char *id;
 	const char *unit;
 } map[] = {
-	{ "current",	"A" },
-	{ "power",	"W" },
-	{ "temp",	"°C" },
-	{ "voltage",	"V" },
-	{ 0, },
+	{ "current", "A" },
```

Comment:
curios: is there an option to somehow align table columns?
we can leave it for later;

Reply:
https://clang.llvm.org/docs/ClangFormatStyleOptions.html

there are alot of align options - I can try to figure it out.

Reply:
I poked around, and It doesn't look like it...

we can add /* clang-format off */ & /* clang-format on */ and that is about it....